ts/modules/sunburst.src.ts

```diff
@@ -202,7 +202,13 @@ var CenteredSeriesMixin = H.CenteredSeriesMixin,
     rad2deg = 180 / Math.PI,
     seriesTypes = H.seriesTypes,
     setTreeValues = mixinTreeSeries.setTreeValues,
-    updateRootId = mixinTreeSeries.updateRootId;
+    updateRootId = function (series: any): string {
```

Comment:
How does this differ from the treemap series? In stead of an override, can we come up with a general solution in the mixin itself? I don't have the answer, just asking... Maybe the same issue also applies to treemap series, but we can't see it?

Reply:
I analyzed the problem again and it turned out that my previous solution is not right.
The problem here is that the sunburst chart looks exactly the same on the level 0 (represented in code as `“”`) and level 1 but only if there’s only one element on level 1.
If there are at least two children on level 1 everything works fine: http://jsfiddle.net/BlackLabel/8vt5rqy7/ 

So, I think that the fix here is to handle the scenario when there’s only 1 element on level 1. Here’s a demonstration version of that fix - hiding traverse up button: http://jsfiddle.net/BlackLabel/0dv35mfr/
```
    if (rootId === '' ||
      (series instanceof H.seriesTypes.sunburst &&
        rootId === series.tree.children[0].id && series.tree.children.length === 1)) {
```
Of, course the code should be placed in sunburst module (not treemap) so some refactoring of `renderTraverseUpButton `method is needed.

The blinking effect you mentioned is caused by the fact that the sunburst switches between the level 0 and 1 (and they both look the same).

Reply:
Thanks! Is this ready to merge now? It seems that the PR was marked ready for review, but the mentioned changes are not committed. Did you forget to push?

Reply:
Not ready yet. Just wanted to make sure that my idea is good.  I'm going to commit probably today or tomorrow.

Reply:
Yes I think the idea is good. But maybe, for consistency, we can have the same situation in tree maps? What if we have a tree map with 1 item on level 1? Maybe it is enough just to look for the number of items on level 1, regardless of the series type?